### DISCUSSION TITLE
UI Datepicker

### DISCUSSION THREAD

**Discussion by DaThorstholm** [datalust/seq-tickets#2082] [Ideas]
If you have choosen a custom date, then add To and From date fields so it's easier to edit the chosen date. 
There is way to many clicks to edit the date. For me the old Datepicker was better.

![SeqCustomDate](https://github.com/datalust/seq-tickets/assets/71385216/bc2be0e2-a660-4bd6-8a07-ed70b69c18b7)

**Comment by nblumhardt** [comment]
Thanks for the note @DaThorstholm. This is https://github.com/datalust/seq-tickets/issues/1975, which we're planning to address in the next minor release.

Best regards,
Nick
